redwood.toml

```diff
@@ -7,11 +7,11 @@
 
 [web]
   title = "Redwood App"
-  port = 8924
-  apiUrl = "/.redwood/functions" # you can customise graphql and dbauth urls individually too: see https://redwoodjs.com/docs/app-configuration-redwood-toml#api-paths
+  port = 8929
+  apiUrl = "/.netlify/functions"
```

Comment:
Why do we need this ?

Reply:
I have an error with the port number with command `yarn rw dev`